Add redis based adapter (#25)
joshleblanc/view_component_reflex: README.md

end
```

## Existing Fast Redis based State Adapter

This adapter uses hmset and hgetall to reduce the number of operations. 

> This is also the recommended way if you using it AnyCable.

Reviewer: ```suggestion
This is the recommended adapter if you are using AnyCable.
```
